operators/tutorials/authentication/README.md

```diff
@@ -17,6 +15,18 @@ Both components can be protected via auth and even be configured independently.
 
 Additionally, all other Pinot components such as **pinot-server** and **pinot-minion** can be configured to authenticate themselves to pinot-controller via the same mechanism. This can be done independently of (and in addition to) using 2-way TLS/SSL to ensure intra-cluster authentication on the lower networking layer.
 
-### Quickstart
+## Quickstart
+
+If you'd rather dive directly into the action with an all-in-one running example, we provide an **AuthQuickstart** runnable with Apache Pinot. This sample app is preconfigured with the settings below but only intended as a dev-friendly, local, single-node  deployment.
```

Comment:
```suggestion
If you'd rather dive directly into the action with an all-in-one running example, we provide an **AuthQuickstart** runnable with Apache Pinot. This sample app is preconfigured with the settings below but intended only as a dev-friendly, local, single-node  deployment.
```

Reply:
I didn't make any changes to this file, but am happy to add the edit.